build-tools/automation/azure-pipelines.yaml

```diff
@@ -999,6 +1000,7 @@ stages:
       inputs:
         artifactName: Test Results - Designer - Windows
         targetPath: $(Build.ArtifactStagingDirectory)/designer-binlogs
+      condition: always()
```

Comment:
An earlier incarnation of this PR was breaking the designer tests. I forgot to add this to the commit message:
```
* [build] designer tests should always upload artifacts, even if tests fail
```